docs/dev/clusteroperator.md

```diff
@@ -140,55 +131,34 @@ When your operator begins rolling out a new version it must continue to report t
 
 For instance, if the CVO is working towards 4.0.1 and has already successfully deployed 4.0.0, the conditions might be reporting:
 
-* `Failing` is false with no message
+* `Degraded` is false with no message
 * `Available` is true with message `Cluster has deployed 4.0.0`
 * `Progressing` is true with message `Working towards 4.0.1`
 
 If the controller reaches 4.0.1, the conditions might be:
 
-* `Failing` is false with no message
+* `Degraded` is false with no message
 * `Available` is true with message `Cluster has deployed 4.0.1`
 * `Progressing` is false with message `Cluster version is 4.0.1`
 
 If an error blocks reaching 4.0.1, the conditions might be:
 
-* `Failing` is true with a detailed message `Unable to apply 4.0.1: could not update 0000_70_network_deployment.yaml because the resource type NetworkConfig has not been installed on the server.`
+* `Degraded` is true with a detailed message `Unable to apply 4.0.1: could not update 0000_70_network_deployment.yaml because the resource type NetworkConfig has not been installed on the server.`
 * `Available` is true with message `Cluster has deployed 4.0.0`
 * `Progressing` is true with message `Unable to apply 4.0.1: a required object is missing`
 
-The progressing message is the first message a human will see when debugging an issue, so it should be terse, succinct, and summarize the problem well.  The failing message can be more verbose. Start with simple, easy to understand messages and grow them over time to capture more detail.
-
-
-#### Conditions and Install/Upgrade
-
-Conditions determine when the CVO considers certain actions complete, the following table summarizes what it looks at and when.
-
-
-| operation | version | available | degraded | progressing | 
-|-----------|---------|-----------|----------|-------------|
-| Install completion[1] | current(whatever was being installed) | true | any | any
-| Begin upgrade | any | any | any | any
-| Begin upgrade (w/ force) | any | any | any | any 
-| Upgrade completion[2]| newVersion(target version for the upgrade) | true | false | false
-
-[1] Install works on all components in parallel, it does not wait for any component to complete before starting another one.
-
-[2] Upgrade will not proceed with upgrading components in the next runlevel until the previous runlevel completes.
-
-See also: https://github.com/openshift/cluster-version-operator/blob/a5f5007c17cc14281c558ea363518dcc5b6675c7/pkg/cvo/internal/operatorstatus.go#L176-L189
```

Comment:
why did we remove this section ?

Reply:
Because it's covered in the reconciliation doc.  Was there anything here you think is not reflected there?